ethereumj-core/src/main/java/org/ethereum/net/rlpx/EncryptionHandshake.java

```diff
@@ -360,11 +360,11 @@ public ECPoint getRemotePublicKey() {
             return token;
         }
 
-        public SHA3Digest getIngressMac() {
+        public KeccakDigest getIngressMac() {
             return ingressMac;
         }
 
-        public SHA3Digest getEgressMac() {
+        public KeccakDigest getEgressMac() {
             return egressMac;
         }
     }
```

Comment:
That's correct. Even the yellowpaper was updated to reflect this.
It's confusing now that not everybody has renamed it yet, I admit. 

Eventually all references to SHA3 should disappear... even the operation name in the VM, Solidity and Web3.js, ... but I don't think everybody is ready for that yet.

I purposely didn't change the method name `sha3(..)` to `kec(..)` and `kec512(...)` to keep the change less controversial, but I'd be happy to see that change.